test-utils/src/main/kotlin/Utils.kt

```diff
@@ -28,7 +28,7 @@ import java.time.Instant
 val DEFAULT_ANALYZER_CONFIGURATION = AnalyzerConfiguration(false, false)
 val DEFAULT_REPOSITORY_CONFIGURATION = RepositoryConfiguration()
 
-val TIMESTAMP_REGEX = Regex("(timestamp): \".*\"")
+val TIMESTAMP_REGEX = Regex("(end_time|start_time|timestamp): \".*\"")
```

Comment:
Nit: Would it be nicer to list `start_time` first here, as I believe alphabetical sorting does not make that much sense in this case?

Reply:
Done.